test/integration/metrics/prometheus_test.go

```diff
@@ -0,0 +1,974 @@
+// Copyright (c) F5, Inc.
```

Comment:
I think we have done integrations two different ways if you compare these tests to the ones here https://github.com/nginx/agent/blob/v3/test/integration/dataplane_api_test.go. Think we need to have a discussion on what we want define as unit test or an integration tests.

Reply:
Yeah I think you're right, I always get confused about the testing terminology 😅 Now that I think about it, maybe my tests are more like component tests rather than integretaion tests 🤔 

We'll discuss it at the office anyway 👌

Reply:
We agreed that I will move the content of these tests to the unit tests, as integration tests should deploy in Docker both the Agent and the app it is integrating with.